packages/@react-spectrum/button/docs/ActionButton.mdx

```diff
@@ -73,7 +85,12 @@ function Example() {
   let [count, setCount] = React.useState(0);
 
   return (
-    <ActionButton icon={<Edit />} onPress={() => setCount(c => c + 1)}>{count} Edits</ActionButton>
+    <ActionButton onPress={() => setCount(c => c + 1)}>
```

Comment:
I kind of want to keep this broken so that we fix it. Remove the icon from this one example.